pkg/models/task_search.go

```diff
@@ -102,103 +170,46 @@ func convertFiltersToDBFilterCond(rawFilters []*taskFilter, includeNulls bool) (
 			continue
 		}
 
-		if f.field == "reminders" {
-			filter, err := getFilterCond(&taskFilter{
-				// recreating the struct here to avoid modifying it when reusing the opts struct
-				field:      "reminder",
-				value:      f.value,
-				comparator: f.comparator,
-				isNumeric:  f.isNumeric,
-			}, includeNulls)
-			if err != nil {
-				return nil, err
+		subTableFilterParams, ok := subTableFilters[f.field]
+		if ok {
+			// in fact, it's better to skip all unstrict filters when we're filtering with subquery.
```

Comment:
Is this a todo? Because the comment seems to be different from the code around it.

Reply:
Removed comment. It's not valid.